index.html

```diff
@@ -22,13 +22,18 @@
       <nav class="mobile-menu">
         <h1 class="logo">My Logo</h1>
         <img class="navMenu" src="images/hamburger.png" id="trigger" alt="menu icon" />
+        <ul id="info-text">
+        <li class="text-1">Portfolio</li>
+         <li class="text-1">About</li>
+        <li class="text-1">Contact</li>
+        </ul>
       </nav>
     </header>
       <section id="landing">
        
-        <div class="hello"><h2>I’m Francis, Glad to see you!</h2></div>
+        <div class="hello"><h2>I’m Francis<br> Glad to see you!</h2></div>
         <div class="landingText">
-          <p>
+          <p class="text-3">
```

Comment:
- [ ] As requested by the previous reviewer please stick to the design as much as possible. Please align the `let's connect and `social media icons`. See the image below for reference
- [ ] please provide the same `font` and `font weight` to the `let's connect`
### Your design
![kyour](https://user-images.githubusercontent.com/81727988/194683887-9c898ee7-9149-456e-9844-1ca5fe5437b4.png)
### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/194683896-ad9521e0-cdef-4ac5-919f-aad503fdfac7.png)